packages/common/src/hooks/useUSDCBalance.ts

```diff
@@ -31,22 +31,24 @@ export const useUSDCBalance = () => {
     [dispatch]
   )
 
-  useEffect(() => {
-    const fetch = async () => {
-      setStatus(Status.LOADING)
-      try {
-        const account = await getUserbankAccountInfo(audiusBackend, {
-          mint: 'usdc'
-        })
-        const balance = (account?.amount ?? new BN(0)) as BNUSDC
-        setData(balance)
-        setStatus(Status.SUCCESS)
-      } catch (e) {
-        setStatus(Status.ERROR)
-      }
+  const refresh = useCallback(async () => {
+    if (status === Status.LOADING) return
```

Comment:
I think you have to take `status` as a dependency now?

Reply:
yeah lint helped me